services/worker/tests/job_runners/split/test_descriptive_statistics.py

```diff
@@ -662,6 +664,34 @@ def test_list_statistics(
     assert computed == expected
 
 
+@pytest.fixture
+def struct_thread_panic_error_parquet_file(tmp_path_factory: pytest.TempPathFactory) -> str:
+    repo_id = "__DUMMY_TRANSFORMERS_USER__/test_polars_panic_error"
+    hf_api = HfApi(endpoint=CI_HUB_ENDPOINT)
+
+    dir_name = tmp_path_factory.mktemp("data")
+    hf_api.hf_hub_download(
+        repo_id=repo_id,
+        filename="test_polars_panic_error.parquet",
+        repo_type="dataset",
+        local_dir=dir_name,
+        token=CI_USER_TOKEN,
+    )
+    return str(dir_name / "test_polars_panic_error.parquet")
+
+
+def test_polars_struct_thread_panic_error(struct_thread_panic_error_parquet_file: str) -> None:
+    from polars import Float64, List, Struct
+    from polars import Utf8 as String  # string type in polars <0.20 is called Utf8, in 0.20 it's an alias to String
```

Comment:
Now that we use 0.20, we can just import String, right?

Reply:
I don't know. I understand that it was for 0.19, but now that we use 0.20, it's not needed anymore, right?